src/app/component/comments/components/comments-list/comments-list.component.html

```diff
@@ -2,8 +2,13 @@
       class="comment-body-wrapper"
       [ngClass]="dataType === 'comment' ? 'wrapper-comment': 'wrapper-reply'">
   <div class="comment-avatar">
-    <img src="assets/img/comments/comment_author.png"
-         alt="comment-author">
+    <!-- <img src={{comment.author.userProfilePicturePath}}
+         alt="comment-author"> -->
```

Comment:
Why do you leave this commented out block of code?

Reply:
Just missed it and forgot to delete. Not needed actually:)